main.tf

```diff
@@ -0,0 +1,210 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+locals {
+  default_tags = {
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+
+  acm_certificate_arn = "${var.acm_certificate_arn == "" ? module.aws-cert.acm_certificate_arn : var.acm_certificate_arn}"
+  distribution_label  = "${var.project}-${var.environment}-distribution_label"
+}
+
+module "vpc" {
```

Comment:
What the purpose of VPC in this module?

Reply:
One of requirements is reusing existed security groups, and I have no idea how to link security group with cloudfront and s3

Reply:
removed vpc and security groups